deps/v8/tools/mb/docs/user_guide.md

```diff
@@ -20,7 +20,7 @@ For more discussion of MB, see also [the design spec](design_spec.md).
 
 ### `mb analyze`
 
-`mb analyze` is responsible for determining what targets are affected by
+`mb analyze` is reponsible for determining what targets are affected by
```

Comment:
which repo this typos should be reported?

Reply:
In the V8 project: https://bugs.chromium.org/p/v8/issues/list